packages/contractkit/src/wrappers/Accounts.test.ts

```diff
@@ -86,4 +86,15 @@ testWithGanache('Accounts Wrapper', (web3) => {
     const sig = await getParsedSignatureOfAddress(account, signer)
     await accountsInstance.authorizeValidatorSignerAndBls(signer, sig, newBlsPublicKey, newBlsPoP)
   })
+
+  test('SBAT set the wallet address to the caller', async () => {
+    await accountsInstance.createAccount()
+    await accountsInstance.setWalletAddress(accounts[0])
+  })
+
+  test('SBAT set the wallet address to a different wallet address', async () => {
```

Comment:
A test that it is not possible to set the wallet address without the PoP would be good.

Reply:
Added